docs/src/examples/components/Input/index.tsx

```diff
@@ -3,9 +3,11 @@ import * as React from 'react'
 import Rtl from './Rtl'
```

Comment:
these temporary changes are introduced for the sake of example to reviewer - won't be merged